packages/server/lib/models.ts

```diff
@@ -17,6 +17,7 @@ export interface ProviderTemplate {
     authorization_url: string;
     authorization_params?: Record<string, string>;
     scope_separator?: string;
+    default_scope?: string;
```

Comment:
That's a really cool idea, but we should support multiple scopes here. Let's change this to an array everywhere.